dist/common/scripts/scylla_raid_setup

```diff
@@ -37,8 +37,8 @@ if __name__ == '__main__':
                         help='specify disks for RAID')
     parser.add_argument('--raiddev', default='/dev/md0',
                         help='MD device name for RAID')
-    parser.add_argument('--update-fstab', action='store_true', default=False,
-                        help='update /etc/fstab for RAID')
+    parser.add_argument('--enable-on-nextboot', action='store_true', default=False,
+                        help='mount RAID on next boot')
     parser.add_argument('--root', default='/var/lib/scylla',
```

Comment:
Please keep --update-fstab for compatibility. add_argument supports ['--enable-on-nextboot', '--update-fstab'].

Reply:
Actually without the brackets.